cobc/flag.def

```diff
@@ -120,10 +120,11 @@ CB_FLAG (cb_flag_winmain, COB_SHOW, "winmain",
 
 #ifdef HAVE_ATTRIBUTE_CONSTRUCTOR
 CB_FLAG_ON (cb_flag_use_constructor, 1, "use-constructor",
+	_("  -fuse-constructor     generate internal one-time code via constructor"))
 #else
 CB_FLAG (cb_flag_use_constructor, 0, "use-constructor",
-#endif
 	_("  -fuse-constructor     generate internal one-time code via constructor"))
+#endif
```

Comment:
thank you for inspecting this - please use the `COB_SHOW` variant as in the item above instead

Reply:
Question though : the two branches do not use the same macro (`CB_FLAG_ON` vs `CB_FLAG`) ; is this intentional ?

Reply:
yes, we enable it by default and show it in the help if it is supported, otherwise disable by default and hide from the help

Reply:
So if the two branches must use different macros, I can't really proceed as you did with `COB_SHOW`, right ?

Reply:
Should be okay now.